elastic/kibana
# [Synthetics] refresh data on visualization scrubbing

## Summary

Resolves https://github.com/elastic/kibana/issues/156776

Refreshes data when visualization scrubbing occurs.

This also has the nice benefit of refreshing data when the monitor is changed, or the location is changed, which I think is a nice touch.

[Synthetics-Monitor-History-Synthetics---Kibana (1).webm](https://user-images.githubusercontent.com/11356435/236366374-9d0b4213-048a-498b-b9a8-3fbaeb0c1602.webm)

### Testing

1. Create a monitor and wait for some data to fill in
2. Scrub a visualization on the history or errors page
3. Notice the page updates
